python/lsst/daf/persistence/policy.py

```diff
@@ -437,7 +437,16 @@ def dump(self, output):
         :return:
         """
         data = copy.copy(self.data)
-        yaml.dump(data, output)
+        keys = ['defects', 'needCalibRegistry', 'levels', 'defaultLevel', 'defaultSubLevels',
```

Comment:
Please add a comment before this list to explain that you are specifically handling a subset of the allowed keys to do something special with them and then will dump the remainder in the standard way.